docs/source/getting_started/using_entitysets.ipynb

```diff
@@ -0,0 +1,337 @@
+    "\n",
+    "If we look at the schema from the transactions dataframe and the new sessions dataframe, we see two more operations that were performed automatically:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "es[\"transactions\"].ww.schema"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "es[\"sessions\"].ww.schema"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "1. It removed \"device\", \"customer_id\", \"zip_code\" and \"join_date\" from \"transactions\" and created a new columns in the sessions dataframe. This reduces redundant information as the those properties of a session don't change between transactions.\n",
+    "2. It copied and marked \"session_start\" as a time index column into the new sessions dataframe to indicate the beginning of a session. If the base dataframe has a time index and ``make_time_index`` is not set, ``normalize_dataframe`` will create a time index for the new dataframe. In this case it would create a new time index called \"first_transactions_time\" using the time of the first transaction of each session. If we don't want this time index to be created, we can set ``make_time_index=False``.\n",
+    "\n",
+    "If we look at the dataframes, we can see what ``normalize_dataframe`` did to the actual data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "es[\"sessions\"].head(5)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "es[\"transactions\"].head(5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To finish preparing this dataset, create a \"customers\" dataframe using the same method call."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "es = es.normalize_dataframe(\n",
+    "    base_dataframe_name=\"sessions\",\n",
+    "    new_dataframe_name=\"customers\",\n",
+    "    index=\"customer_id\",\n",
+    "    make_time_index=\"join_date\",\n",
+    "    additional_columns=[\"zip_code\", \"join_date\"],\n",
+    ")\n",
+    "\n",
+    "es"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Using the EntitySet\n",
+    "\n",
+    "Finally, we are ready to use this EntitySet with any functionality within Featuretools. For example, let's build a feature matrix for each product in our dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "feature_matrix, feature_defs = ft.dfs(entityset=es, target_dataframe=\"products\")\n",
```

Comment:
`target_dataframe` is being updated to `target_dataframe_name`

If you want to see if this cell will work properly, you can create a test branch and merge your changes here with the code in the `ww-cfm-update` WIP branch. I think the code there should be updated enough that you can get back a feature matrix.

Reply:
Updated in f0940a2d. The notebook was able to run end-to-end with the changes from `ww-cfm-update`